client/blocks/login/index.jsx

```diff
@@ -233,7 +233,7 @@ class Login extends Component {
 			postHeader = (
 				<p className="login__header-subtitle">
 					{ translate(
-						'Your account will enable you to start using the features and benefits offered by Jetpack & WooCommerce Services.'
+						'Your Jetpack account will enable you to start using the benefits offered by Jetpack & WooCommerce Services.'
```

Comment:
Humm, I wonder if it's correct to say "Jetpack account" instead of "WordPress.com account"? I get it's prolly to lessen brand confusion but wordpress.com will be there no matter what (in the URL if not elsewhere). What if they already have one but don't realize that now with this copy changed?

Reply:
Yup, I agree. I originally changed these in the WC flow to match the mocks, but I think it would be better to use the same language (and it's already translated). Fixed.